caps.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const faker = require('faker')
+const events = require('./lib/event.js');
+const { address } = require('faker');
+
```

Comment:
I can't run `node caps.js` because your package.json is in another folder so it can't find the correct file path